backend/src/routes/v0/tool.py

```
@@ -67,54 +83,19 @@ async def get_arcade_tools(
     name: str = Query(default="", description="Name of the tool to get"),
     limit: int = Query(default=25, description="Limit the number of tools to get"),
     offset: int = Query(default=0, description="Offset the number of tools to get"),
-    type: Literal[
-        "static", 
-        # "formatted", 
-        "scheduled"
-    ] = Query(default="static", description="Type of tools to get"),
-    # user: ProtectedUser = Depends(verify_credentials),
+    type: Literal["static", "scheduled"] = Query(default="static", description="Type of tools to get"),
     db: AsyncSession = Depends(get_async_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)